### ISSUE TITLE
The font file url in @fontface rule doesn't seem to follow convention

### ISSUE THREAD

**Issue by UzEE** [choffmeister/roboto-fontface-bower#13]
I've also run into an issue similar to #7 (closed, so I made a new one). I'm using ```grunt``` and ```bower``` in my build system and my structure is like

```
root/
|__scripts/
|__styles/
|__fonts/
|__images/
|__views/
|__index.html
```

All existing packages I've seen and using (bootstrap, font-awesome, winjs etc.) follow the convention of including the font from ```../fonts/``` instead of ```fonts/``` like this package is doing. They all generally structure their bower package internally as follows:

```
root/
|__bower_components/
   |__font-awesome/
      |__css/
      |  |__style files.css
      |__js/
      |  |__script files.css
      |__...
      |__fonts/
      |  |__font files.ttf
      |__...
      |__bower.json
```

The build system (```grunt```) in this case is setup to copy font files from the ```bower_components/package/fonts/...``` directory to ```dist/fonts/...``` directory, and everything fine. In ```roboto-fontface``` case however, the css files look for the fonts in ```dist/styles/fonts/...``` instead which don't exist.

I'd suggest that adopting the more widely accepted convention would be best (I haven't encountered this sort of hierarchy before so I not qualified to say which is the more widely accepted version), so that integration headaches could be avoided.

**Comment by choffmeister** [comment]
I agree to your argument of not being in line with other projects. Can you please create a PR for the reordering of the file structure?

**Comment by choffmeister** [comment]
Fixed with 372fa6e695d59b7b14b1ed714557f6a2e9825aab
